src/Standards/Squiz/Tests/Commenting/InlineCommentUnitTest.php

```diff
@@ -52,6 +52,8 @@ public function getErrorList($testFile='')
                 126 => 2,
                 130 => 2,
                 149 => 1,
+                196 => 1,
```

Comment:
This is an unexpected change. The new error for this line is:
` 196 | ERROR | [x] There must be no blank line following an inline comment (Squiz.Commenting.InlineComment.SpacingAfter)`
-- which is the same as (one of) the error(s) thrown on line 97.

What's the value of keeping old line 196? Should we replace it with the new test? If not, let's reword the comment to explain its value/use.

Reply:
@fredden Oh darn! Too much copy/paste, not enough reading. Sorry about that.

> What's the value of keeping old line 196? Should we replace it with the new test? If not, let's reword the comment to explain its value/use.

The new test should have been placed _above_ that test, which is a specific "comment at the end of a file" test. This is actually annotated in a separate comment just above it (line 193-196 in the new diff).

My bad for not paying enough attention. Fixed up now by moving the new test to be above the "end of file" test.

Thank you for asking the right question ;-)